src/switch_ivr_play_say.c

```diff
@@ -1929,6 +1986,10 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_play_file(switch_core_session_t *sess
 				break;
 			}
 		}
+		if (sp_prev) { 
+			switch_safe_free(sp_prev);
+			sp_prev = NULL;
```

Comment:
you don't have to do this, switch_safe_free(sp_prev) would set sp_prev to NULL after it frees it.

Reply:
Agreed, this was redundant, a simple `switch_safe_free(sp_prev);` is enough. This is no longer an issue since the sp_prev was moved to static storage.